src/evaluation/retriever/retriever_eval_squad.py

```diff
@@ -147,13 +169,25 @@ def single_run(parameters):
     if retriever_type in ["sbert", "dpr"]:
         document_store.update_embeddings(retriever, index=doc_index)
 
+    if epitca_perf_file:
```

Comment:
Do we have to use this ? can't we use the questions directly from the knowledge base ? 
In the case we do not want to use the comparison with Eptica one need to be reminded to remove the file in the parameters otherwise, the questions from eptica will be asked.

Reply:
We need this to feed the latest questions and answers given by the CNIL. As we talked about, we can later integrate these questions with the rest of the CNIL knowledge base.

When not using the comparison with Epitca, the epitca_perf_file option can be set to `[None]` in the config file.